### DISCUSSION TITLE
OG-Core developer meeting, Dec. 20, 2021

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#765] [Meeting]
**Agenda for OG-Core developer meeting: 12/20/21**
noon-1pm ET
Join Zoom Meeting
https://zoom.us/j/97619464215?pwd=UlU5VzkzWGN2VUZsd2k2MWt2L0dXUT09

Meeting ID: 976 1946 4215
Passcode: 903621
One tap mobile
+13017158592,,97619464215#,,,,*903621# US (Washington DC)
+13126266799,,97619464215#,,,,*903621# US (Chicago)

This meeting is open to the public.

**Agenda**
* OG-Core Government capital/infrastructure PR #722 progress
* OG-UK demographics [PR #3](https://github.com/PSLmodels/OG-UK/pull/3) progress
* OG-UK steady-state computation
* @rickecon debt and deficits piece progress
* Miscellaneous OG-Core/OG-USA PR's and Issues

Next meeting: Monday, Jan. 3, 2022, noon-1pm ET

@jdebacker @MaxGhenis @jpycroft @nikhilwoodruff

**Comment by jpycroft** [comment]
Hello @rickecon @jdebacker @MaxGhenis @nikhilwoodruff,

Apologies, but I will miss the meeting on Monday. I feel bad, because I confidently said two weeks ago that I could make it on the 20th. In the end, I'll be flying. (Originally, we were due to fly on Thursday 23rd. Now in an attempt to get ahead of omicron, Elisabetta and the girls fly tomorrow, and I'll follow them on Monday evening.) Sorry about that. 

And in case I'm not in touch before: Merry Christmas!
